### PR TITLE
This is a blog post about upcoming logging features for Camel K.

### PR DESCRIPTION
**Pull request by orpiske**
Any thoughts or suggestions?

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: content/blog/2021/05/new-camel-k-logging-features/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Additional log levels can be turned on or off at ease, using the logging.level trait. Users
+can inform the minimum log level (i.e.: `TRACE`, `DEBUG`, `INFO`, etc) and control how much
+information is presented by the logger.
+
+
+## Custom logging format
+
+Leveraging the previous features, it's also possible to control the format of the message
+displayed. Since the runtime leverages Camel Quarkus and, by extension, Quarkus itself, all
+the logging format options are available to customize the log messages.
```
Comment (aldettinger) [inline content/blog/2021/05/new-camel-k-logging-features/index.md:41]: So here, we are talking about -t logging.format ?

File: content/blog/2021/05/new-camel-k-logging-features/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+parts of Camel K and the integrations it generated.
+
+During the 1.5 development release we dedicated some time to review the logging capabilities
+of Camel K. This version introduces a new [logging trait](https://camel.apache.org/camel-k/latest/traits/logging.html) that simplifies access to the logging configuration available on the runtime.
+
+## Colorized logs
+
+By default, Camel K will colorize the log output, making it easier to distinguish between
+log levels, loggers, timestamps and the logging message. Naturally, we understand that not
+everyone likes colorized logs. As such it's possible to turn it by using the logging.color
```
[the remark this answers is not in the document]
Comment (orpiske) [reply content/blog/2021/05/new-camel-k-logging-features/index.md:22]: +1

File: content/blog/2021/05/new-camel-k-logging-features/index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+## Easier support for additional debug levels
+
+Additional log levels can be turned on or off at ease, using the logging.level trait. Users
+can inform the minimum log level (i.e.: `TRACE`, `DEBUG`, `INFO`, etc) and control how much
+information is presented by the logger.
+
+
+## Custom logging format
+
+Leveraging the previous features, it's also possible to control the format of the message
```
[the remark this answers is not in the document]
Comment (orpiske) [reply content/blog/2021/05/new-camel-k-logging-features/index.md:39]: +1, I think "displayed message" makes it clearer.

File: content/blog/2021/05/new-camel-k-logging-features/index.md (hunk 1 above)
Comment (orpiske) [reply content/blog/2021/05/new-camel-k-logging-features/index.md:41]: I am trying to explain that the logging format options are available (i.e.: `%d{xxx} %H %m`, etc that we can use to customize the logging fields.

Comment (orpiske) [reply content/blog/2021/05/new-camel-k-logging-features/index.md:41]: I think I'll rephrase this...

Comment (orpiske) [reply content/blog/2021/05/new-camel-k-logging-features/index.md]: I like the idea about adding an example. I'll include that!

**Review by aldettinger — APPROVED** [review]
Looks good to me.

**Comment by orpiske** [comment]
Thanks, I unmarked it as a draft so we can merge it at an appropriate time.

**Comment by zregvart** [comment]
I've rebased this on the recent changes we made to fix issues with HTML validation we had. I want to make sure we don't introduce a new issue with this. Not very likely that it will but just to be sure. When the checks successfully complete I'll merge this.

File: content/blog/2021/05/new-camel-k-logging-features/index.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+summary: "New Camel K Logging Features for Camel K 1.5"
+---
+
+
+As part of our continuous work to improve Camel K and improve its stability, features and
+usefulness to our community, we recently worked on logging improvements for different
+parts of Camel K and the integrations it generated.
+
+During the 1.5 development release we dedicated some time to review the logging capabilities
+of Camel K. This version introduces a new [logging trait](https://camel.apache.org/camel-k/latest/traits/logging.html) that simplifies access to the logging configuration available on the runtime.
```
Comment (zregvart) [inline content/blog/2021/05/new-camel-k-logging-features/index.md:16]:
```suggestion
of Camel K. This version introduces a new [logging trait](/camel-k/latest/traits/logging.html) that simplifies access to the logging configuration available on the runtime.
```

We need to go with relative links, the reason is that we don't check remote links and if we leave links to camel.apache.org as remote this link won't get checked and we would not know it's broken.

Comment (orpiske) [reply content/blog/2021/05/new-camel-k-logging-features/index.md:16]: Thanks Zoran! Let's do this.

**Comment by orpiske** [comment]
Preview looks almost ok. I found a small rendering problem I'd like to fix.

**Comment by orpiske** [comment]
Now it looks good.

**Review by oscerd — APPROVED** [review]

**Pull request merged**